src/components/ElementOption.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+type PropsElementOption = {
+  label: string;
+  value: string;
+  onChange: Function;
+};
+function ElementOption(props: PropsElementOption) {
+  const handleChangeInput = (e: React.FormEvent<HTMLInputElement>) => {
+    props.onChange(e.currentTarget.value);
+  };
+  return (
+    <>
+      <label style={{ display: "flex", flexDirection: "row" }}>
+        <span style={{ marginRight: 5, fontSize: 14, minWidth: 40 }}>
+          {props.label}:
+        </span>
+        <input
+          onChange={handleChangeInput}
```

Comment:
The size input can be a number type, to increase with arrow keys.

Reply:
@paulomenezes Done => replace with select box